### PR TITLE
Edits for clarity:  aria-brailleroledescription

### PR DESCRIPTION
**Pull request by estelle**
Reread the page and made some edits along the way. No major content changes.

### CODE REVIEW

File: files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md [hunk 1]
```diff
@@ -9,11 +9,15 @@ tags:
...
+Braille is not a one-to-one transliteration of letters and numbers, but rather it includes various abbreviations, contractions, and characters that represent words (known as lolograms). 
+
+Instead of converting long role descriptions to Braille, the `aria-brailleroledescription` allows for providing an abbreviated version of the [`aria-roledescription`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription) value, which is a human-readable, author-localized description for the role of an element, for improved user experience with Braille interfaces.
```
Comment (sideshowbarker) [inline files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md:18]:
```suggestion
Instead of converting long role descriptions to Braille, the `aria-brailleroledescription` attribute allows for providing an abbreviated version of the [`aria-roledescription`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription) value, which is a human-readable, author-localized description for the role of an element, for improved user experience with Braille interfaces. 
```

File: files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md [hunk 2]
```diff
@@ -9,11 +9,15 @@ tags:
...
+Braille is not a one-to-one transliteration of letters and numbers, but rather it includes various abbreviations, contractions, and characters that represent words (known as lolograms). 
+
+Instead of converting long role descriptions to Braille, the `aria-brailleroledescription` allows for providing an abbreviated version of the [`aria-roledescription`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription) value, which is a human-readable, author-localized description for the role of an element, for improved user experience with Braille interfaces. 
+
+Basically, the value of `aria-brailleroledescription` is an abbreviated version of the `aria-roledescription` to be converted into Braille.
```
Comment (sideshowbarker) [inline files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md:20]:
```suggestion
Instead of converting long role descriptions to Braille, the `aria-brailleroledescription` attribute allows for providing an abbreviated version of the [`aria-roledescription`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription) value, which is a human-readable, author-localized description for the role of an element, for improved user experience with Braille interfaces. 

Basically, the value of `aria-brailleroledescription` is an abbreviated version of the `aria-roledescription` attribute to be converted into Braille.  
```

File: files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md [hunk 3]
```diff
@@ -24,23 +28,27 @@ Braille is not a one-to-one transliteration of letters and numbers, but rather i
...
 
-Only use of `aria-brailleroledescription` to clarify the purpose of non-interactive container roles like group or region, or to providing a more specific description of a widget in a braille context. As the `aria-brailleroledescription` property overrides how assistive technologies localize and express the name of a role in Braille, inappropriate values will prevent users from understanding and interacting with an element on braille interfaces.
+The `aria-brailleroledescription` attribute should only be used to clarify the purpose of non-interactive container roles like group or region, or to provide a more specific description of a widget in a braille context. 
+
+As the `aria-brailleroledescription` property overrides how assistive technologies localize and express the name of a role in Braille, inappropriate values will prevent users from understanding and interacting with an element on braille interfaces.
```
Comment (sideshowbarker) [inline files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md:33]:
```suggestion
The `aria-brailleroledescription` attribute should only be used to clarify the purpose of non-interactive container roles like "group" or "region", or to provide a more specific description of a widget in a braille context. 

As the `aria-brailleroledescription` attribute overrides how assistive technologies localize and express the name of a role in Braille, inappropriate values will prevent users from understanding and interacting with an element on braille interfaces.
```

File: files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md [hunk 4]
```diff
@@ -24,23 +28,27 @@ Braille is not a one-to-one transliteration of letters and numbers, but rather i
...
 
-Only use `aria-brailleroledescription` if the [`aria-roledescription`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription) is present; but if the `aria-roledescription` value works in Braille, the Braille version of the attribute is not needed. In general, `aria-brailleroledescription` should only be used in the rare case when a `aria-roledescription` is too verbose for Braille.
+Only use `aria-brailleroledescription` if the [`aria-roledescription`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription) is present. However, if the `aria-roledescription` value works in Braille, the Braille version of the attribute is not needed. In general, `aria-brailleroledescription` should only be used in the rare case when a `aria-roledescription` is too verbose for Braille.
```
Comment (sideshowbarker) [inline files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md:35]:
```suggestion
Only use `aria-brailleroledescription` if [`aria-roledescription`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription) is present. However, if the `aria-roledescription` value works in Braille, the Braille version of the attribute is not needed. In general, `aria-brailleroledescription` should only be used in the rare case when a `aria-roledescription` is too verbose for Braille.
```

File: files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md [hunk 5]
```diff
@@ -24,23 +28,27 @@ Braille is not a one-to-one transliteration of letters and numbers, but rather i
...
 
 A few rules to remember:
 
-- Only apply `aria-brailleroledescription` to elements with a valid ARIA role or has with implicit role semantics.
+- Only apply `aria-brailleroledescription` to elements with a valid ARIA role or elements with implicit role semantics.
 - The `aria-brailleroledescription`, if present, must have a non-empty, none null value that is different from the `aria-roledescription` value, which, in turn, is different from the ARIA explicit or role or implicit semantic role.
-- Avoid using Unicode Braille Patterns. If they must be used, ensure the `aria-brailleroledescription` value doesn't contain only characters in Unicode Braille Patterns and contains content other than white space and Braille Pattern dots-0. If the content is only in Unicode Braille Patterns, the value will not be translated according to the user's preferred translation table.
+- Avoid using Unicode Braille Patterns. If they must be used, ensure the `aria-brailleroledescription` value contains other content other than Unicode Braille Patterns, white-space, and Braille Pattern dots-0.
```
Comment (sideshowbarker) [inline files/en-us/web/accessibility/aria/attributes/aria-brailleroledescription/index.md:41]:
```suggestion
- Avoid using Unicode Braille Patterns. If they must be used, ensure the `aria-brailleroledescription` value contains  content other than Unicode Braille Patterns, whitespace, and Braille Pattern dots-0. 
```

**Review by sideshowbarker — APPROVED** [review]
